rudder-inventory-ldap/SOURCES/rudder-inventory-ldap.init

```diff
@@ -174,6 +174,7 @@ BACKUP_SUFFIX="`date +%Y%m%d%H%M%S`.ldif"
 # Other
 TIMEOUT="60"    # Max time to stop process
 FD_LIMIT="2048" # Max file descriptor
+VM_LIMIT="108490720" # Max virtual memory
```

Comment:
Yes, but if i understand correctly, this value is the maximum virtual memory allowed to be associated  to the process, which includes mapped files.

So this has to be dependent or far above mdb size (ie unlimited).
So you could compute it again, or simply put unlimited here